tests/lax_numpy_test.py

```diff
@@ -2912,6 +2912,19 @@ def testLinspace(self, start_shape, stop_shape, num, endpoint,
         self._CompileAndCheck(jnp_op, args_maker,
                               check_dtypes=False, atol=tol, rtol=tol)
 
+  @parameterized.named_parameters(
+      jtu.cases_from_list(
+        {"testcase_name": "_dtype={}".format(dtype),
+         "dtype": dtype,
+         "rng_factory": rng_factory}
+        for dtype in number_dtypes
+        for rng_factory in [jtu.rand_default]))
+  def testIssue3014(self, dtype, rng_factory):
```

Comment:
Though there are test cases with analogous names throughout the codebase, nowadays I try to have more descriptive names than `testIssueNNNN` and instead have a comment with a URL pointing to the issue. Consider that an optional suggestion!

Reply:
Done